python/version_details/version_details.py

```diff
@@ -520,6 +520,10 @@ def load_data(self, entity):
             self.clear()
             return
 
+        # Ensure project schema will be loaded. The project schema is needed
+        # to reference Version fields in various UI and other elements.
+        shotgun_globals.run_on_schema_loaded(lambda : None, entity.get("project"))
```

Comment:
@thebeeland  and @gomezll , Great points. Yeah it's been several months now so I'm not sure exactly the reasoning for this specific function call.
The odd thing is if I remember correctly, the bug only occurred when there is *no cache* so I'm surprised this would have fixed it if the cache isn't available yet so it should be async.
@gomezll I will send you the Jira with the reproduction steps and review the code a bit more.